packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -100,6 +106,69 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [onMouseUp, hideDropSelector],
   );
 
+  const handleKeyDown = React.useCallback(
+    event => {
+      switch (
+        event.keyCode // Begin switch 1
+      ) {
+        case 190: // "." (dot) key - Moves to the next element
+        case 188: // "," (comma) key - Moves to the previous element
+          // if (event.ctrlKey) { // Begin if 1
+          virtualCursorElements[vcIndex].classList.remove('virtual-focused');
+          setVcIndex(index => {
+            const step = event.keyCode === 190 ? 1 : -1;
+            const nextIndex = index + step;
+
+            // If nextIndex is bigger than number of elements move it to 0
+            // If nextIndex is smaller than 0 move it to the lastElement
+            // Otherwise move to the nextIndex
+            const newIndex =
+              nextIndex >= virtualCursorElements.length
+                ? 0
+                : nextIndex < 0
+                ? virtualCursorElements.length - 1
+                : nextIndex;
+
+            virtualCursorElements[newIndex].classList.add('virtual-focused');
+            setFocusedVcElement(virtualCursorElements[newIndex]);
+            return newIndex;
+          }); // End setVcIndex
+          // } // End if 1
+          return;
+        case 13: // Enter key - Clicks the current element
+          virtualCursorElements[vcIndex].click();
+          return;
+        case 121: // F10 key
+          if (event.shiftKey) {
+            // Begin if 1
+            const mouseEvent = document.createEvent('MouseEvents');
+            mouseEvent.initMouseEvent(
```

Comment:
`initMouseEvent` is deprecated please use the constructor instead `new MouseEvent()`

More info here: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent

Reply:
Actually, the current code for showing the context menu does not work. So resolved it for now by removeing the code. The possibility to show a context menu on an element focused with the virtual cursor is not essential. Anyway, we did some investigation with Milan and couldn't find a way how to dispatch an event which would open a context menu. [Here is a JSFiddle](https://jsfiddle.net/kolaps33/6jtufpxq/49/) which we used for testing.